conformance/fhir-ig-davinci-hrex/src/test/java/com/ibm/fhir/ig/davinci/hrex/test/ProfileTest.java

```diff
@@ -53,4 +62,23 @@ public void testUSCoreValidation() throws Exception {
             throw e;
         }
     }
+
+    @Factory
```

Comment:
how did it run before?

Reply:
nm, now i see ProfileTestFactory.java (which is now removed). :-)

Reply:
yeah - if it's outside the class it looses the scope